OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -265,11 +265,10 @@ public void onEventMainThread(DiscussionThreadUpdatedEvent event) {
 
     @SuppressWarnings("unused")
     public void onEventMainThread(DiscussionCommentPostedEvent event) {
-        // If a new comment was posted in a listed thread, increment its comment count
+        // If a new response/comment was posted in a listed thread, we need to update the list
         for (int i = 0; i < discussionPostsAdapter.getCount(); ++i) {
             final DiscussionThread discussionThread = discussionPostsAdapter.getItem(i);
             if (discussionThread.containsComment(event.getComment())) {
-                discussionThread.incrementCommentCount();
```

Comment:
Incrementing on the responses screen takes affect on this screen's thread as well, since that's the one we passed. So, we only need to call `notifyDataSetChanged` here.